### PR TITLE
MusicXML: \mark export

### PR DESCRIPTION
**Pull request by FelippeRoza**
First attempt to support \mark feature.
By now it is only able to export the following syntax: '\mark \ default' and '\mark' and add marks from 'A' to 'Z'.

### CODE REVIEW

File: ly/musicxml/ly2xml_mediator.py [hunk 1]
```diff
@@ -353,6 +354,17 @@ def new_key(self, key_name, mode):
...
+    def new_mark(self):
+        if self.bar is None:
+            self.new_bar()
+        if self.bar.has_music():
+            new_bar_attr = xml_objs.BarAttr()
+            new_bar_attr.set_mark(self.current_mark)
+            self.add_to_bar(new_bar_attr)
+        else:
+            self.current_attr.set_mark(self.current_mark)
+        self.current_mark = chr(ord(self.current_mark) + 1)
```
Comment (FelippeRoza) [inline ly/musicxml/ly2xml_mediator.py]: Currently this solution just adds marks correctly from `A` to `Z` (I wasn't aware of the `I` skipping). I'm working to make it able to add `AA` after `Z` and so on.

Comment (PeterBjuhr) [inline ly/musicxml/ly2xml_mediator.py]: Great!

Comment (FelippeRoza) [inline ly/musicxml/ly2xml_mediator.py]: Now it is better. It correctly skips `I` and changes to the correct mark sequence after an ending `Z`. Now it just needs to support the integer argument to set the mark manually.

**Comment by FelippeRoza** [comment]
Now it supports an integer argument to set the mark manually. The mark.ly test file comprehends the `I` skipping, the `Z` to `AA` and the `AZ` to `BA` transitions.

File: ly/musicxml/xml_objs.py [hunk 2]
```diff
@@ -453,11 +455,11 @@ def has_music(self):
         return False
 
     def has_attr(self):
-        """ Check if bar contains attribute. """
-        for obj in self.obj_list:
-            if isinstance(obj, BarAttr):
-                return True
-        return False
+         """ Check if bar contains attribute. """
```
Comment (PeterBjuhr) [inline ly/musicxml/xml_objs.py]: Please take a look at this before I merge!

**Pull request merged**